Decred Wallet/Features/Settings/ServerSetTableViewController.swift

```diff
@@ -43,7 +43,7 @@ class ServerSetTableViewController: UITableViewController {
             self.navigationController?.popViewController(animated: true)
             return
         } else {
-            self.showMessage(title: "Invalid input", userMessage: "please input a valid IP address", buttonTitle: "ok")
+            self.showMessage(title: LocalizedStrings.invalidInput, userMessage: LocalizedStrings.inputValidServer, buttonTitle: LocalizedStrings.ok)
```

Comment:
How is `LocalizedStrings.inputValidIP` different from `LocalizedStrings.inputValidServer`?